pika/adapters/select_connection.py

```diff
@@ -75,16 +75,6 @@ def _adapter_disconnect(self):
             self.ioloop.remove_handler(self.socket.fileno())
         super(SelectConnection, self)._adapter_disconnect()
 
-    def _flush_outbound(self):
-        """Call the state manager who will figure out that we need to write
-            then call the poller's poll function to force it to process events.
-        """
-        self._manage_event_state()
-        # Force our poller to come up for air, but in write only mode
-        # write only mode prevents messages from coming in and kicking off
-        # events through the consumer
-        self.ioloop.poll(write_only=True)
-
 
 class IOLoop(object):
```

Comment:
@wjps, interestingly, after all these changes, it seems like IOLoop class is no longer needed, and could be replaced by a function that simply creates the appropriate poller.